pom.xml

```diff
@@ -64,13 +64,13 @@
 		<dependency>
 			<groupId>org.spigotmc</groupId>
 			<artifactId>spigot-api</artifactId>
-			<version>1.10.2-R0.1-SNAPSHOT</version>
```

Comment:
Weird, it looks like you forked an earlier release, and are PR'ing against latest. I'll resolve locally.